react-common/components/share/ShareInfo.tsx

```diff
@@ -92,6 +94,19 @@ export const ShareInfo = (props: ShareInfoProps) => {
         else setShareState("share")
     }
 
+    const handlePublishInVscodeClick = async () => {
+        setShareState("publishing");
+        let publishedShareData = await publishAsync(name, thumbnailUri, isAnonymous);
+        setShareData(publishedShareData);
+        if (!publishedShareData?.error) {
+            setShareState("publish");
+
+            pxt.tickEvent(`share.openInVscode`);
+            window.open(vscodeDevUrl + publishedShareData.url.split("/").pop(), "_blank");
```

Comment:
Would it make sense to add some kind of `shareCode` field to ShareData that can do the `publishedShareData.url.split("/").pop()` "behind the scenes", so to speak? (probably needs a better name than shareCode, though)

Reply:
yes! ...but i'm going to merge this as-is for now since that would be a bigger change (and i need this for a demo tomorrow)